charts/node-analyzer/templates/runtimeScanner/eveconnector-api-deployment.yaml

```diff
@@ -0,0 +1,84 @@
+{{- if and (not (include "deploy-nia" .)) .Values.nodeAnalyzer.deploy (not .Values.gke.autopilot) .Values.nodeAnalyzer.runtimeScanner.deploy .Values.nodeAnalyzer.runtimeScanner.eveConnector.deploy }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "nodeAnalyzer.fullname" . }}-eveconnector-api
+  labels:
+    app.kubernetes.io/name: {{ include "nodeAnalyzer.name" . }}-eveconnector
+{{ include "nodeAnalyzer.labels" . | indent 4 }}
+spec:
+  replicas: {{ .Values.nodeAnalyzer.runtimeScanner.eveConnector.settings.replicas }}
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
```

Comment:
let's remove this line for now

Reply:
question: is this talking about the same thing as the one in Sysdig charts? naming looks a bit different because the other one is under updateStrategy instead of strategy.

Reply:
@xiao-sysdig this one is part of the runtimeScanner, and it's a deployment which is why the spec looks a bit different. I think we shouldn't make any changes of our own here

Reply:
I see. thanks